src-gui/src/renderer/components/pages/swap/swap/done/BitcoinPunishedPage.tsx

```diff
@@ -21,11 +37,93 @@ export default function BitcoinPunishedPage({
             <br />
             We tried to redeem the Monero with the other party's help, but it
             was unsuccessful (reason: {state.content.reason}). Attempting again
-            at a later time might yield success. <br />
+            at a later time might yield success.
+            <br />
           </>
         )}
       </DialogContentText>
       <FeedbackInfoBox />
+      <ManualCoopRedeemModal
+        open={modalOpen}
+        onClose={() => setModalOpen(true)}
+      />
     </>
   );
 }
+
+interface ManualCoopRedeemModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+function ManualCoopRedeemModal({ open, onClose }: ManualCoopRedeemModalProps) {
+  const [success, setSuccess] = useState<boolean | null>(null);
+  const [key, setKey] = useState("");
```

Comment:
Never use empty string in state, always use `useState<string | null>` and explicitly set to null

Reply:
Ok in this case it's okay because these are actually text fileds